package.json

```diff
@@ -5,19 +5,29 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "format": "prettier --write '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "lint": "next lint && prettier --check '**/?(.)*.{md,css,scss,js,ts,tsx,json,yaml,yml}'",
+    "test": "jest --watch"
   },
   "dependencies": {
-    "next": "12.0.3",
+    "happy-dom": "^2.24.5",
+    "next": "^12.0.7",
```

Comment:
Could you set `NEXT_TELEMETRY_DISABLED=1` in CI to disable Next.js telemetry there?